lib/ex_unit/lib/ex_unit.ex

```diff
@@ -243,6 +242,9 @@ defmodule ExUnit do
       different modules run in parallel. It defaults to `System.schedulers_online * 2`
       to optimize both CPU-bound and IO-bound tests;
 
+    * `:max_failures` - maximum number of tests that may fail. When the maximum is reached, the
+       execution of tests is stopped and the results are returned.
```

Comment:
Minor nitpick: looking at other items, I think this line should end with a semicolon `;` for consistency (and not with a dot `.`)